### DISCUSSION TITLE
"Unsubscribe" button  doesn't seem to work in drop-down menu

### DISCUSSION THREAD

**Discussion by Kokhirus** [FakeFishGames/Barotrauma#14085] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

"Unsubscrube" button doesn't seem working.
![image](https://github.com/FakeFishGames/Barotrauma/assets/91697233/39e6f4ee-0faa-4994-b800-d36951fcb030)

### Reproduction steps

1. Open Mods Tab
2. Right-click on any mod
3. Click "Unsubscribe"

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested, working correctly. Closing.
